libs/core/src/lib/select/select.component.ts

```diff
@@ -57,525 +96,362 @@ export interface OptionStatusChange {
-    get value(): any {
-        return this._selectProxy.value$.value;
+    set value(newValue: any) {
+        if (newValue !== this.internalValue) {
+            this.writeValue(newValue);
+            this.internalValue = newValue;
+        }
     }
 
-    /** Whether the select is in compact mode. */
+    /**
+     * Whether the select is in compact mode.
+     */
     @Input()
     compact = false;
 
-    /** Whether option components contain more than basic text. */
+    /**
+     * Glyph to add icon in the select component.
+     */
     @Input()
-    extendedBodyTemplate = false;
+    glyph = 'slim-arrow-down';
 
-    /** Max height of the popover. Any overflowing elements will be accessible through scrolling. */
-    @Input()
-    maxHeight: string;
+    /**
+     * If the option should be unselected and value changed to undefined, when the current value is
+     * not presented in option array. Switching it off can be handy, when there is some delay between providing
+     * possible options and chosen value.
+     */
 
-    /** Glyph to add icon in the select component. */
+    /**
+     * Binds to control aria-labelledBy attribute
+     */
     @Input()
-    glyph = 'slim-arrow-down';
+    ariaLabelledBy: string = null;
 
-    /** Whether close the popover on outside click. */
+    /**
+     * Sets control aria-label attribute value
+     */
     @Input()
-    closeOnOutsideClick = true;
+    ariaLabel: string = null;
 
-    /** Popper.js options of the popover. */
+    /**
+     * Max height of the popover. Any overflowing elements will be accessible through scrolling. *
+     */
     @Input()
-    popperOptions: PopperOptions = {
-        placement: 'bottom-start',
-        modifiers: {
-            preventOverflow: {
-                enabled: true,
-                escapeWithReference: true,
-                boundariesElement: 'scrollParent'
-            }
-        }
-    };
+    maxHeight: string;
 
     /**
-     * Preset options for the Select body width, whatever is chosen, the body has a 600px limit.
-     * * `at-least` will apply a minimum width to the body equivalent to the width of the control. - Default
-     * * `equal` will apply a width to the body equivalent to the width of the control.
-     * * 'fit-content' will apply width needed to properly display items inside, independent of control.
+     * Time to wait in milliseconds after the last keydown before focusing or 
+     * selecting option based on alphanumeric
+     * keys.
      */
     @Input()
-    fillControlMode: PopoverFillMode = 'at-least';
+    typeaheadDebounceInterval = 250;
+
+    /** Select Input Mobile Configuration */
+    @Input()
+    mobileConfig: MobileModeConfig = { hasCloseButton: true };
 
     /** Custom template used to build control body. */
     @Input()
     controlTemplate: TemplateRef<any>;
 
-    /** The element to which the popover should be appended. */
     @Input()
-    appendTo: ElementRef;
```

Comment:
I guess this can be considered as a breaking change, becuase this Input has been removed